implementations.md

```diff
@@ -30,6 +30,7 @@ Here are a list of implementations that live in Fantasy Land:
 * [Sanctuary](https://github.com/plaid/sanctuary) is a refuge from unsafe JavaScript. It provides FL-compatible Either and Maybe types.
 * [Fluture](https://github.com/Avaq/Fluture) is a high-performance monadic alternative to Promises.
 * [Ramda Adjunct](https://github.com/char0n/ramda-adjunct) is a community-maintained extension of Ramda
+* [Folktale](https://folktale.origamitower.com/) implements Maybe, Result, Validation Task and Future as FL compatible Monads.
```

Comment:
I think there's a comma missing between “Validation” and “Task”. Also, “FL-compatible” should be hyphenated in this context.